src/CountUp.js

```diff
@@ -200,4 +195,95 @@ class CountUp extends Component {
   }
 }
 
+const NO_ELEMENT = -1;
+
+const decorator = (f, action) => (...args) => {
+  const result = f(...args);
+  action(result);
+  return result;
+};
+
+const isFunction = f => typeof f === 'function';
+
+const isValid = prop => prop !== undefined && prop !== null;
+
+const removeInvalidProps = obj =>
+  Object.keys(obj).reduce(
+    (acc, curr) => (isValid(obj[curr]) ? { ...acc, [curr]: obj[curr] } : acc),
+    {},
+  );
```

Comment:
I see the benefits here but I think this would disallow users to "reset" certain props. Maybe let's just use the original props and override defaults?

Reply:
We could argue with ignoring `undefined` but I'd say in the above's example we could just programmatically leave the props we don't want to pass to the hook. So let's keep the implementation simple